firebase-storage/src/test/java/com/google/firebase/storage/PathingTest.java

```diff
@@ -164,7 +164,7 @@ public void dotdotTest() throws Exception {
         FirebaseStorage.getInstance()
             .getReferenceFromUrl("gs://benwu-test1" + ".storage.firebase.com/child/../");
 
-    assertEquals(ref.getName(), "..");
+    assertEquals("..", ref.getName());
```

Comment:
I have to look up this ordering, every. single. time. (FTR, you've done the correct thing.)

However, if you're going to go to the effort of fixing this, you might consider migrating to assertThat instead. It makes it much more obvious.

Optional.

Reply:
Keeping it as is to match the base CL.